README.md

```diff
@@ -38,8 +38,16 @@ You have two options:
 *   Install globally `eslint` and plugins
 
     *   `$ npm i -g eslint [eslint-plugins]`
-    *   Activate `Use Global Eslint` package option
+    *   Activate `Use Global Eslint` package option in Sublime's config
+    ```
+      "linter-eslint":
+        "useGlobalEslint": true
+    ```
     *   (Optional) Set `Global Node Path` with `$ npm config get prefix`
+    ```
+      "linter-eslint":
+        "globalNodePath": "/usr/local"
```

Comment:
This is a _perfect_ example as to how putting copyable examples in the readme is a bad idea. This may be valid for your system, but other *NIX systems may place it in a different folder, and that isn't even a valid path on Windows systems.